packages/@react-spectrum/table/src/TableView.tsx

```diff
@@ -193,6 +195,7 @@ function TableView<T extends object>(props: SpectrumTableProps<T>, ref: DOMRef<H
     if (reusableView.viewType === 'headerrow') {
       return (
         <TableHeaderRow
+          onHoverChange={setHeaderRowHovered}
```

Comment:
Should this be hovered and focused? I'm asking if the visual indicator of resizable columns also makes sense for the header with keyboard focus. Because when some columns don't resize there is no visual indicator of which columns resize, so guess by pressing enter. Currently, I'd never know a table had resizable columns with keyboard navigation. I also noticed all the header column resizers also show when keyboard is resizing one column.

Reply:
There will be a menu indicator eventually, so this should only be for hover. Keyboards will access through the menu.